support-models/src/main/scala/com/gu/support/workers/states/PaymentMethodDisplayFields.scala

```diff
@@ -0,0 +1,98 @@
+package com.gu.support.workers.states
+
+import cats.syntax.functor._
+import com.gu.support.encoding.Codec
+import com.gu.support.encoding.Codec.deriveCodec
+import com.gu.support.workers.states.WireFields.{DirectDebitWirePaymentMethodDisplayFields, PaymentMethodDisplayFieldsWire, JustTypeWirePaymentMethodDisplayFields}
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder, HCursor, Json}
+
+sealed trait PaymentMethodDisplayFields{
+  def `type`: String
+}
+
+sealed trait JustTypePaymentMethodDisplayFields extends PaymentMethodDisplayFields
+
+case object CreditCardDisplayFields extends JustTypePaymentMethodDisplayFields{
+  override val `type` = "CreditCardReferenceTransaction"
+}
+case object PaypalDisplayFields extends JustTypePaymentMethodDisplayFields{
+  override val `type` = "Paypal"
+}
+case class DirectDebitDisplayFields(
+                                          bankAccountNumberMask: String,
```

Comment:
Minor: the indentation looks a bit strange here

Reply:
yeah Intellij insists in formatting it like that. I'll fix it